src/components/MobileControls/MobileControlSheet/MobileControlSheet.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {memo, useCallback} from 'react';
+import {List, Sheet} from '@gravity-ui/uikit';
+import block from 'bem-cn-lite';
+
+import {ListItem} from '../../../models';
+
+import './MobileControlSheet.scss';
+
+const b = block('dc-mobile-control-sheet');
+
+export interface MobileControlSheetProps {
```

Comment:
It would be better if the type definitions were after or before, and not between, the variable declarations.